src/LottoCalculator.js

```diff
@@ -0,0 +1,74 @@
+import { Console } from '@woowacourse/mission-utils';
+import { OUTPUT_MESSAGE } from './lib/constant.js';
+
+class LottoCalculator {
+  #result = {
+    three: 0,
+    four: 0,
+    five: 0,
+    fiveBonus: 0,
+    six: 0,
+  };
+
+  #ROI = 0;
+
+  constructor(buyPrice, lottos, winningNumbers, bonusNumber) {
+    this.buyPrice = buyPrice;
+    this.lottos = lottos;
+    this.winningNumbers = winningNumbers;
+    this.bonusNumber = bonusNumber;
+    this.calculateResult(lottos, winningNumbers, bonusNumber);
+    this.calculateROI(buyPrice);
+  }
+
+  calculateResult(lottos) {
+    lottos.forEach((lotto) => {
+      const matchCount = this.calculateMatchCount(lotto);
+      if (matchCount === 3) this.#result.three += 1;
+      if (matchCount === 4) this.#result.four += 1;
+      if (matchCount === 5 && !this.matchBonusNumber(lotto))
+        this.#result.five += 1;
+      if (matchCount === 5 && this.matchBonusNumber(lotto))
+        this.#result.fiveBonus += 1;
+      if (matchCount === 6) this.#result.six += 1;
+    });
+  }
+
+  calculateMatchCount(lotto) {
+    return lotto.filter((number) => this.winningNumbers.includes(number))
+      .length;
+  }
+
+  matchBonusNumber(lotto) {
+    if (lotto.includes(parseInt(this.bonusNumber, 10))) {
+      return true;
+    }
+    return false;
```

Comment:
```suggestion
    return lotto.includes(parseInt(this.bonusNumber, 10));
```
이렇게 축약할 수 있을 것 같습니다 😄